scripts/generators/csv_generator.py

```diff
@@ -1,33 +1,10 @@
-# Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I suspect this header was removed by your editor, but it would need to remain with any changes.